src/transformers/trainer.py

```diff
@@ -2723,6 +2726,18 @@ def _issue_warnings_after_load(self, load_result):
                 f"There were unexpected keys in the checkpoint model loaded: {load_result.unexpected_keys}."
             )
 
+    def _evaluate(self, trial, ignore_keys_for_eval):
```

Comment:
```suggestion
    def _evaluate(self, trial, ignore_keys_for_eval, skip_scheduler=False):
```
We need to disable scheduler skipping, otherwise the scheduler will be skipped on the first step!